contents/docs/cdp/common-questions.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Troubleshooting and FAQs
+---
+
+## My app isn't working, what do I do?
+
+1. Checking that the app is enabled with the correct configuration options in the [data pipeline tab](https://us.posthog.com/apps). You can find the correct configuration options in the [app's documentation](/docs/cdp) by searching for it in the destinations, transformations, or filtering sections of the docs.
+2. Click "Logs & metrics" and go to the `processEvent metrics` tab to check that the app is processing events without errors.
+3. Check the `Logs` tab to see if there are any errors.
+4. Go to the data management tab to check if there any [ingestion warnings](https://us.posthog.com/data-management/ingestion-warnings).
+5. If the app relates to an external service, check that the external service is working correctly. Make a request to the relevant API endpoint or use a tool like [webhook.site](https://webhook.site/).
```

Comment:
you'd use webhook.site as the destination from PostHog to check that PostHog side sends the events as expected